packages/kit/src/utils/platform.js

```diff
@@ -0,0 +1,3 @@
+export function non_node_runtime() {
+	return typeof globalThis.Deno !== 'undefined' || typeof globalThis.Bun !== 'undefined';
+}
```

Comment:
what about this? I always try to avoid negatives in function names, particularly if the return value needs to be negated, resulting in a double negation. Also we can add a check for Node 18.11 if we give it a more generic name

```suggestion
export function should_polyfill() {
  if (typeof Deno !== 'undefined') return false;
  if (typeof Bun !== 'undefined') return false;
  const [major, minor] = process.versions.node.split('.');
  return +major < 18 || +major === 18 && +minor < 11;
}
```

Reply:
I'm happy with that as long as we're okay with it being unclear what version of undici users will end up with per @Conduitry's point: https://github.com/sveltejs/kit/pull/8338#issuecomment-1371572551

Reply:
Yeah, would be equally happy with this

```suggestion
export function should_polyfill() {
	if (typeof Deno !== 'undefined') return false;
	if (typeof Bun !== 'undefined') return false;
	return true;
}
```